.travis.yml

```diff
@@ -105,6 +114,8 @@ script:
     if [[ $BUILD_DOCS == false ]]; then
       export MPL_REPO_DIR=$PWD  # needed for pep8-conformance test of the examples
       gdb -return-child-result -batch -ex r -ex bt --args python tests.py --processes=$NPROC --process-timeout=300 $TEST_ARGS
+      # py.test --pep8 --cov=matplotlib `python -c "import os ; import inspect ; import matplotlib; print( os.path.dirname(inspect.getfile(matplotlib)) )"`
+      # py.test --cov=matplotlib --pyargs matplotlib
```

Comment:
I *think* these comment lines are preventing the error code from returning to Travis, so the tests are in fact failing, but Travis isn't aware of it and marking them as passed.  It seems odd the comments would have a return value, but I can't think of anything else (and I'm not much of a bash expert).